src/views/Editor.vue

```diff
@@ -224,6 +220,7 @@ main.editor {
         display: flex;
         flex-direction: column;
         position: relative;
+        border-right: 1px solid rgba(255, 255, 255, 0.2);
```

Comment:
That shouldn't be there, was testing out something else. Will probably make another PR for that and other CSS changes.